test/lib/formCache.test.ts

```diff
@@ -1,14 +1,20 @@
+import { objectContaining } from "expect/build/asymmetricMatchers";
 import { v4 as uuidv4 } from "uuid";
 import {
   CacheEntry,
   FormCacheFactory,
   IFormCache,
 } from "../../src/lib/formCache";
+import { Registration } from "../../src/lib/model/registration.model";
 
 describe("FormCache", () => {
   let cache: IFormCache;
   let id: string;
 
+  const emptyCache = {
+    registration: expect.any(Registration),
+  };
+
```

Comment:
This is dodgy.  Cache may not be `empty` if the `Registration` object has things in it.